forms/GroupedDropdownField.php

```diff
@@ -84,6 +87,9 @@ public function Field($properties = array()) {
 		return FormField::create_tag('select', $this->getAttributes(), $options);
```

Comment:
Okay, once your pull request is merged I'll adjust it to make sure its in line...